sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -3,12 +3,7 @@
 ### 4.3.0b4 (Unreleased)
 
 #### Features Added
-
-#### Breaking Changes
-
-#### Bugs Fixed
-
-#### Other Changes
+- Added support for AAD authentication for the sync client
```

Comment:
Why only sync for now?

Reply:
I'm splitting it into two PR's to not make it too many lines